examples/run_aims.py

```diff
@@ -56,17 +54,11 @@ def test_run_aims():
         else:
             assert (type(sockets_calc.calc) == Aims)
 
+    # Test to make sure that we correctly handle scenario when environment variable isn't
+    # set in ASE 3.23. This presents issues downstream, so environment must be set 
+    # i.e. executable and species directory.
+    with self.assertRaises(KeyError):
+                del os.environ['ASE_AIMS_COMMAND']
+                get_aims_calculator(0) 
```

Comment:
@GaryLZW : I'm not sure why 0 is needed as an input to get_aims_calculator?

Reply:
Ah. This is for the `dimension` positional argument. It has no default value

> def get_aims_calculator(dimensions, spin=None, relativistic=None, k_grid=None, xc="pbe", compute_forces="true", directory='./', **kwargs):